README.md

```diff
@@ -180,8 +200,8 @@ system user apache uses.  It's easy just to do this by hand first. :)
 
 If you want to use apache, add apache config file to sites-enabled and edit it
 
-        sudo cp ../apache/lp /etc/apache2/sites-available/lp
-        vim /etc/apache2/sites-available/lp
+        sudo cp ../apache/lp /etc/apache2/sites-available/lp.conf
```

Comment:
should the template file (lp) be renamed lp.conf? and then this command changed to reflect that?